src/test/java/com/ebivariation/contigalias/api/ContigAliasControllerTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ebivariation.contigalias.api;
+
+import com.ebivariation.contigalias.entities.AssemblyEntity;
+import com.ebivariation.contigalias.entities.ChromosomeEntity;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest
+public class ContigAliasControllerTest {
+
+    private static final String GCA_ACCESSION_HAVING_CHROMOSOMES = "GCA_000003055.3";
+
+    private static final String GCF_ACCESSION_NO_CHROMOSOMES = "GCF_006125015.1";
+
+    @Autowired
+    private ContigAliasController api;
+
+    // TODO replace these tests as persistent db may not have these test cases present
```

Comment:
I don't understand this comment. You can test the controller with a custom DB by providing an AssemblyService bean pre-filled with whatever data you want. The usual thing is to do an inner class Config with a @Bean method that returns the AssemblyService. Either that or an AssemblyService mock.

Reply:
This is an old comment that I had added when the controller could only get from db and not the server. I have removed it and re-enabled the tests.